public/index.html

```diff
@@ -186,6 +186,150 @@ <h6 class="card-title text-center py-1" style="background-color: black;">Creator
+                <div class="alert alert-info form-inline col-sm-10" role="alert">
+                    <div id="#"></div>
+                </div>
+            </div>
+
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="input">Assignee</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Creator</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Estimated Time</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="input">Category</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Priority</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Difficulty</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="inputState">Status</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- log work-------------------------------------------------------- -->
+
+    <!--my=margin in y axis, mx=margin in x axis, ml=marginLeft, mr= marginRight, mt=marginTop, mb=marginBottom ,
+         Similarly for padding py,px,pl,pr,pt,pb
+        and w=width  -->
+
+    <div class="container py-5 px-0">
+        <div class="bg-dark py-2 px-5">
+            <h3 class="mt-5 fontHeading text-white">Log Work</h3>
+            <div class="row pt-5 pb-2 ml-1">
+                <div class="alert alert-info form-inline col-sm-4" role="alert">
+                    <div style="font-weight: bold;" class="mr-3">Id:</div>
```

Comment:
Please convert all the margins into padding if possible

Reply:
ok